code/modules/admin/sql_notes.dm

```diff
@@ -239,7 +239,7 @@
 			output += "<a href='?_src_=holder;shownoteckey=[index_ckey]'>[index_ckey]</a><br>"
 			CHECK_TICK
 		qdel(query_list_notes)
-		message_admins("The note search started by [usr.ckey] has complete. CPU should return to normal.")
+		message_admins("The note search started by [usr.ckey] has completed. CPU should return to normal.")
```

Comment:
Perhaps `Note search started by [usr.ckey] complete. CPU should return to normal.`?